src/lua/fiber.c

```diff
@@ -822,6 +929,8 @@ static const struct luaL_Reg lbox_fiber_meta [] = {
 	{"__tostring", lbox_fiber_tostring},
 	{"join", lbox_fiber_join},
 	{"set_joinable", lbox_fiber_set_joinable},
+	{"set_slice", lbox_fiber_set_slice},
+	{"extend_slice", lbox_extend_slice},
```

Comment:
After a discussion with @unera and @alyapunov, we decided to revise the API as follows:
- `set_slice` sets the *current* slice of the current fiber. Does not take `fiber` argument.
- `extend_slice` extends the *current* slice of the current fiber. Does not take `fiber` argument.
- `set_max_slice` sets the *max* slice . If called on a `fiber`, sets the fiber's max slice. If called without `fiber` argument, sets the default max slice for all fibers.

We decided to export the function that sets the current fiber slice (`fiber.set_slice`), because we will need this function anyway in SQL (see my comment about applying the limit to SQL queries). For consistency, with `extend_slice`, we decided to name it `set_slice` and rename the function that sets the max slice to `set_max_slice`. An alternative would be to rename `extend_slice` to `extend_current_slice`, and use `set_current_slice`, but the names would be longer. Besides, without `current` the names looks consistent with `check_slice`.

Please keep C names consistent with Lua to avoid confusion:
- `fiber::max_slice` - max slice.
- `fiber::slice` - current slice, initialized to `fiber::max_slice`.
- `cord::max_slice` - default max slice.

and so on.

Reply:
> fiber::slice - current slice, initialized to fiber::max_slice.
You meant `cord::slice`?

Reply:
Yes. The current fiber slice is reset on each yield so it should probably be stored in struct cord.